README.md

```diff
@@ -32,13 +32,23 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the DOM?
 
+    The DOM or Document Object Model is an API that represents an HTML document as a tree structure comprised of node objects.
+
 2. What is an event?
 
+    Events are actions that take place within a program environment.
```

Comment:
To expand a bit you could say that events are most often triggered by the user (e.g., mouse click, mouse scroll, copying/cutting/pasting something on the page.)